README.md

```diff
@@ -212,6 +212,14 @@ It is possible to interrupt the watcher from another thread with:
 watcher.finish
 ```
 
+#### Get a proxy URL
+You can get a complete URL for connecting a kubernetes entity via the proxy.
+
+```ruby
+client.proxy_url('services', 'srvname', '5001-tcp', 'ns')
```

Comment:
@abonas it's not port and protocol, it's either the port or the port name (ref: https://github.com/kubernetes/kubernetes/blob/release-1.0/pkg/api/v1/types.go#L1112), which by accident in this example was called with the convention `port-protocol`.